.gitpod.yml

```diff
@@ -13,7 +13,7 @@ tasks:
       conda activate pandas-dev
       git pull --unshallow  # need to force this else the prebuild fails
       git fetch --tags
-      python setup.py build_ext -j 4
+      python setup.py build_ext --inplace -j 4
```

Comment:
isn't inplace already in pyproject.toml? just for my understanding, why is this necessary?

Reply:
in https://github.com/pandas-dev/pandas/pull/49917/files, 
```
[build_ext]
inplace = True
```
in setup.cfg was changed to
```
# TODO: Remove after setuptools support is dropped.
[tool.setuptools]
include-package-data = true
```
in pyproject.toml

do they do the same thing? 

the docs don't have `--inplace`

https://github.com/pandas-dev/pandas/blob/bfa793cb684879d73717b77107a9d0685d70d74e/doc/source/development/contributing_environment.rst#L215

so does this need updating there?

Reply:
@lithomas1 sorry for the ping, just trying to understand the above

Reply:
Thanks for the ping. This does indeed seem missing, but the old syntax seems to be distutils specific since I can't find anything on it in the setuptools docs. 
(see https://docs.python.org/3/distutils/configfile.html).

Maybe we can update where we are overriding build_ext in setup.py to also pass through inplace=True as the default?
(here https://github.com/pandas-dev/pandas/blob/11d75d8d17d15d315de6511285d79edeed5c0b73/setup.py#L76-L104?)

I'm not really inclined to change this, so a doc update should suffice (since this time, the meson build work should actually be ready, just waiting on a meson-python release).